charts/kube-prometheus-stack/values.yaml

```diff
@@ -1300,6 +1317,10 @@ kubelet:
     # From kubernetes 1.18, /metrics/resource/v1alpha1 renamed to /metrics/resource
     resourcePath: "/metrics/resource/v1alpha1"
 
+    ## Enable scraping Service Level Indicator (SLI) metrics (/metrics/slis) from kubelet's service
+    ##
+    slis: false
+
     ## MetricRelabelConfigs to apply to samples after scraping, but before ingestion.
     ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig
     ##
```

Comment:
Just to be sure again, cAdvisor permissions are controlled by the `node/metrics` (sub)`resources`, not the `nonResourceURLs` `/metrics/cadvisor`, so it can be gotten even if `/metrics/cadvisor` is removed from `ClusterRole`.

Reply:
I went back and checked the kubernetes source and the path under `/metrics` in kubelet has been controlled by the `node/metrics` (sub)`resources` since auth[nz] was introduced to kubelet in kubernetes v1.5.
Therefore, I believe the `/metrics/cadvisor` in `nonResourceURLs` can be safely removed.